doc/api/vm.md

```diff
@@ -442,6 +446,32 @@ Creating a new `vm.Script` object compiles `code` but does not run it. The
 compiled `vm.Script` can be run later multiple times. The `code` is not bound to
 any global object; rather, it is bound before each run, just for that run.
 
+### script.createCachedData()
+<!-- YAML
+added: REPLACEME
+-->
+
+Creates a code cache that can be used with the Script constructor's
+`cachedData` option. Returns a Buffer. If the code cache cannot be created then
+`ERR_CODE_CACHE_CREATION_FAILED` is thrown. This method may be called at any
```

Comment:
@hashseed What can possibly lead to failure of creating cached data? It would be helpful here for the documentation.